.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Please don't commit this file (git exclude).

Reply:
Removed, but why? It will be present in literally any clone and `node_modules` definitely shouldn't appear in repository.